pyfenix/ws_api.py

```diff
@@ -0,0 +1,71 @@
+"""Implementation of the client API layer for websockets"""
+
+import json
+import logging
+import queue
+from typing import Optional, Tuple
+
+import websockets.client
+
+from pyfenix.api import API, NoConnectionError
+from pyfenix.event import Event
+
+class WebsocketsAPI(API):
+    """Fenix client API using websockets"""
```

Comment:
This makes it sound like you can use the client API without web sockets, which you cannot.

Reply:
Yet.

Reply:
No, never will be able to
HTTP doesn't allow for bidirectionalness

Reply:
Just Fenix client API.  Im talking about the docstring.